kafka/consumer/kafka.py

```diff
@@ -38,6 +38,8 @@
     'auto_commit_interval_messages': None,
     'consumer_timeout_ms': -1,
 
+    'metrics_responder': None,
```

Comment:
the upstream client configuration here is `metric.reporters`. I try to stay consistent, so perhaps `metric_reporters` and support a list of reporter interfaces?